app/views/PaymentHistory.scala.html

```diff
@@ -88,7 +88,7 @@
                     </a>
                     @if(payment.isCredit) {
                     <br>
-                    <a class="govuk-text">
+                    <a class="govuk-text" style="font-size: 16.5px;">
```

Comment:
Instead of setting the font size specifically, we usually use gov classes. This text in the prototype uses "govuk-body-s", so I think it's better if you just replace the class with this, and leave out the style

Reply:
This is just so that, if the gov standard design updates, our pages will also automatically update

Reply:
Made those changes now.